cmudb/tscout/model.py

```diff
@@ -44,6 +44,18 @@ class BPFVariable:
     pg_type: str = None  # Some BPFVariables don't originate from Postgres (e.g., metrics and metadata) so default None
     alignment: int = None  # Non-None for the first field of a struct, using alignment value of the struct.
 
+    # Indicates whether this field is a padding field used to keep user-space/kernel-space structs in alignment.
+    is_padding: bool = False
```

Comment:
Since we assert that `padding_field_size is not None` below, do we need the boolean? Basically,y in the `should_output` function can we just check that `padding_field_size is not None`?